docs/sources/reference/commandline/cli.md

```diff
@@ -1100,6 +1100,10 @@ and Docker images will report:
 
     untag, delete
 
+The `--since` and `--until` parameters can be Unix timestamps, RFC3339
+dates or Go duration strings (e.g. `10m`, `1h30m`) computed relative to
```

Comment:
Is there a reference for Go duration strings we can link to? Docker users are not Go developers. The format looks trivial, but may be useful.